src/web/modules/agreements/states.ts

```diff
@@ -1,14 +1,15 @@
 import {ExternalAgreementState, ExternalTransactionState} from "../../../lib/pay-request/shared";
 
 export enum AgreementListFilterStatus {
-  Created = 'created', Active = 'active', Cancelled = 'cancelled', All = 'all'
+  Active = 'active', All = 'all', Cancelled = 'cancelled', Created = 'created', Inactive = 'inactive'
```

Comment:
I think this is consumed by https://github.com/alphagov/pay-toolbox/blob/master/src/web/modules/agreements/list.njk#L21